src/data/config.rs

```diff
@@ -15,15 +15,24 @@ pub const CONFIG: &str = "config.yaml";
 
 const ADDR: &str = "0.0.0.0";
 const PORT: &str = "3000";
+const DEFAULT_FORGE_API_PAGE_SIZE: u8 = 10;
```

Comment:
It's actually really interesting to consider the different tradeoffs involved here, on one hand, there's just like the traffic aspect of it, at least that's what came to mind for rime.cx, where it seems like generally speaking, we could easily have this to 100 or more (at least to a zero'th approximation). 

But there is really value in limiting it, in case there has never been releases in ages, e.g. as I believe is the case with rocket.rs.

All this kinda gives me the idea that perhaps the age of releases could also play a part somehow. But just like, a thing for later, I really wanna focus on this and #30 first and then if we ever wanna add that extra complexity do it on top of what we get from this and #30.

Reply:
The problem with this approach is that to know the age of a release, you'd have to fetch the release info first. To fetch the release info, you grab the list of releases. So unless you want the very first release, you're now looking at two fetches / request at a minimum. This may work for the `/:user/:repo.tar.gz` endpoint, but it fails for semver, where you want to grab a whole bunch of them release at a time, and the age... doesn't really matter much.

Nevertheless, this is *possible*, in a later PR (not submitted yet, but hinted at in #32) I made the release grabbing a part of the Forge trait, so forges can implement custom ways to fetch releases, as they see fit. That can be adapted to do the kind of fetching you described.

Reply:
Yea I was mostly thinking for the simpler case of `/:user/:repo.tar.gz`, where we could get all we need from a single request (at least for github). E.g. `https://api.github.com/repos/cafkafk/fortune-kind/releases`:

```json
[
  {
    ...
    "tag_name": "v0.1.7",
   ...
    "created_at": "2023-11-02T08:57:37Z",
    "published_at": "2023-11-02T09:01:08Z",
    ...
  },
  {
  ...
  }
```

Here you get multiple releases, so just from that there could potentially be some logic, e.g. if the latest release is older than a year, 6 months, what have you, we are okay with a pre-release.

But again, not something I think we should do now (or even do nescesarrily).